pages/search/search.jsx

```diff
@@ -89,6 +108,17 @@ class Search extends React.Component {
     </div>;
   }
 
+  renderHelpFilter() {
+    let renderedHelpTypes = this.state.helpOptions.map(type => {
+      return <option key={type} value={type}>{type}</option>;
+    });
+
+    return <select id="help-dropdown" className="body-large p-3" value={ this.state.helpType } onChange={  (event) => this.handleHelpChange(event) }>
```

Comment:
Mixing Bootstrap classes and a custom set of targeted styles makes for confusing debugging. Let's stick to targeted styles for this element (but generic as a class- we could re-use this style for other select elements, right?)

Reply:
yep, you're right 💯